CHANGELOG.md

```diff
@@ -22,7 +22,10 @@ We use *breaking* word for marking changes that are not backward compatible (rel
 ### Fixed
 
 ## [v0.27.0](https://github.com/thanos-io/kube-thanos/tree/v0.27.0) (2022-07-07)
-- (no changes from `v0.26.0`)
+
+### Changed
+
+- [#279](https://github.com/thanos-io/kube-thanos/pull/279) Change `hashringConfigmapName` to `hashringConfigMapName` in Receive configuration.
```

Comment:
This needs to go further up in the unreleased section. This section pertains to an already released version of kube-thanos.